.env-sample

```diff
@@ -0,0 +1,3 @@
+SNYK_TOKEN=replaceme
+COVERALLS_REPO_TOKEN=replaceme
```

Comment:
`COVERALLS_REPO_TOKEN` isn't needed locally - there is no need to upload coverage stats unless running on CI